config/locales/en.yml

```diff
@@ -388,7 +388,7 @@ en:
         segment_count: Number of Segments
         subtitle: Subtitle
         title: Title
-        url: Episode URL
+        url: Episode Link
```

Comment:
From what I can see, we use "URL" in many more places than "link", so I would probably change the few "links" to "URL", unless we want to go back and change everything.

<img width="1259" alt="Screenshot 2023-12-28 at 11 47 53 AM" src="https://github.com/PRX/feeder.prx.org/assets/3483/fcd8fc31-684b-4411-8faa-4076a58e5613">